en/core-libraries/caching.rst

```diff
@@ -383,7 +383,7 @@ You can greatly improve the performance of your application by putting results
 that infrequently change, or that are subject to heavy reads into the cache.
 A perfect example of this are the results from
 :php:meth:`Cake\\ORM\\Table::find()`. The Query object allows you to cache
-results using the ``cache()`` method. See the :ref:`caching-query-results` section
+results using the ``cache()`` method. See the :ref:`caching-loaded-results` section
```

Comment:
It didn't jump to the anchor tag after redirect. Shouldn't it be linking to the following anchor tag instead?
https://github.com/cakephp/docs/blob/3.0/en/orm/query-builder.rst#caching-loaded-results

not the current
https://github.com/cakephp/docs/blob/3.0/en/orm/query-builder.rst#caching-query-results

If that's not the case just close it.